modules/core/src/v2/wallet.ts

```diff
@@ -2030,14 +2031,19 @@ export class Wallet {
         reqId: params.reqId,
       });
     } catch (e) {
-      console.error('transaction prebuild failed local validation:', e.message);
-      console.error(
-        'transaction params:',
-        _.omit(params, ['keychain', 'prv', 'passphrase', 'walletPassphrase', 'key', 'wallet'])
-      );
-      console.error('transaction prebuild:', txPrebuild);
-      console.trace(e);
-      throw e;
+      if (this._wallet.multisigType === 'tss') {
+        // TODO: STLX-13411 - implement verify tx for tss wallets
+        console.warn('transaction prebuild failed local validation:', e.message);
+      } else {
+        console.error('transaction prebuild failed local validation:', e.message);
```

Comment:
Is pushing these console.error/warn intentional or just for local testing?

Reply:
this is intentional. right now we don't have verify transaction working with TxRequests.
so i decided to catch the error and output a warning so we can continue developing with TSS wallets.

after we get verify tx working, we will log the error and re-throw.